common/src/main/java/com/microsoft/identity/common/adal/internal/tokensharing/ITokenShareInternal.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.adal.internal.tokensharing;
+
+public interface ITokenShareInternal {
+
+    /**
+     * For the supplied user unique identifier (OID), return the corresponding refresh token for
+     * that account if the current application is a member of FoCI (family of clientIds).
+     *
+     * @param oid The OID of the sought user's FRT.
+     * @return The {@link com.microsoft.identity.common.adal.internal.tokensharing.TokenCacheItem}
+     * serialized to JSON.
+     */
+    String getFamilyRefreshToken(String oid) throws Exception;
```

Comment:
if this is going to support object id and username.... likely should just be identifier.  We may want to support both.

Reply:
ACK - previous implementation supports both `oid` and `upn` -- will update 👍

Reply:
Resolved in:
946147769d0c7c1ba75487243c150e1162ddb4f8
862f13e2166b490ee85a8f089ec73c95596af171